src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.cs

```diff
@@ -85,9 +85,9 @@ private TimeZoneInfo CreateLocal()
                         timeZone = new TimeZoneInfo(
                                             timeZone._id,
                                             timeZone._baseUtcOffset,
-                                            timeZone._displayName,
-                                            timeZone._standardDisplayName,
-                                            timeZone._daylightDisplayName,
+                                            timeZone.DisplayName,
+                                            timeZone.StandardName,
+                                            timeZone.DaylightName,
```

Comment:
Needed to invoke the lazy initialization because `_displayName` would not be populated otherwise.